packages/querygrid/src/components/heatmap/utils.spec.ts

```diff
@@ -36,4 +36,15 @@ describe("HeatMap utils", () => {
         expect(monthSort('jan', 'feb')).toBe(0);
     });
 
+    test ("last12Months", () => {
+        const months = last12Months();
+        expect (months.length).toBe(12);
+
+        // make sure we got 12 unique months
+        const monthNames = Set<string>().asMutable();
```

Comment:
Is usage of asMutable() needed here?

Reply:
Ah shoot.  That doesn't make a lot of sense with a const...